File: sistema veterinario/Mamifero.java
public class Mamifero extends Animal
{
    public Mamifero(String nombre,
                    int edad,
                    double peso,
                    Propietario propietario)
    {
        super(nombre, edad, peso, propietario);
    }

    @Override
    public void aplicarTratamiento()
    {
        System.out.println("Aplicando tratamiento para mamifero.");
    }
}

File: Logica/Mamifero.cs
Mamifero.cs                                                    namespace Sistema_Veterinario.Logica
{
    public class Mamifero : Animal
    {
        public Mamifero(string nombre,
                       int edad,
                       Propietario dueño)
            : base(nombre,
                  edad,
                  dueño)
        {
        }

        public override void AplicarTratamiento()
        {
            Console.WriteLine(
                "Tratamiento para mamíferos");
        }
    }
}